src/org/battleship/Game.java

```diff
@@ -0,0 +1,108 @@
+package org.battleship;
+
+import org.battleship.equipment.Mark;
+import org.battleship.equipment.ShipTypes;
+
+import java.util.Scanner;
+
+import static org.battleship.equipment.Constant.*;
+
+public class Game {
+    private final Battlefield battlefieldPlayer;
+    private final Battlefield battlefieldOpponent;
+    private final Scanner scanner;
+
+    public Game(Battlefield battlefieldPlayer, Battlefield battlefieldOpponent, Scanner scanner) {
```

Comment:
Battlefield should be initialized inside constructor. It is internal details of Game.